kingArchitecture.py

```diff
@@ -114,30 +114,37 @@ def theKing(prompt, openai_api_key):
 
     answers = {}
     color_mapping = {
-        'Llama3': PINK,
-        'MistralAI': CYAN,
-        'Gemini': YELLOW,
-        'Claude': NEON_GREEN,
-        'GPT4o': GOLD
```

Comment:
I don't remember but why did we take out the color mapping for Gpt4o

Reply:
i just did that now, basically gpt4o is the king so right now i just color mapped the king response to orange if you look at lines 167-168:
```
king_content_text = f"**:orange[The King's answer:]**\n\n{king_answer}"
    st.session_state.messages.append({"role": "assistant", "content": king_content_text})
  ```
In the king architecture we never call gpt4o by its name we just call it king so i just hardcoded it for now. However on the duopoly architecture this would have to be changed.